### PR TITLE
i have ensured that human_s and ai_s is string before concating them

### PR DESCRIPTION
**Pull request by kanukolluGVT**
this closes the issue #3077
Does This work?
This is the 1st time i am contributing to opensource
Correct me if i am wrong
Thanks
Screenshot (1096)

### CODE REVIEW

File: langchain/chains/conversational_retrieval/base.py [hunk 1]
```diff
@@ -161,11 +167,7 @@ def from_llm(
         **kwargs: Any,
     ) -> BaseConversationalRetrievalChain:
         """Load chain from LLM."""
-        doc_chain = load_qa_chain(
-            llm,
-            chain_type=chain_type,
-            prompt=qa_prompt,
-        )
+        doc_chain = load_qa_chain(llm, chain_type=chain_type, prompt=qa_prompt,)
```
Comment (vowelparrot) [inline langchain/chains/conversational_retrieval/base.py:170]: These lower changes aren't needed :)

**Review by vowelparrot — COMMENTED** [review]
It would, but there's an issue with our function typing in the first place, which ought to be solved upstream of this. I'll investigate a bit further

File: langchain/chains/conversational_retrieval/base.py (hunk 1 above)
Comment (kanukolluGVT) [reply langchain/chains/conversational_retrieval/base.py:170]: Got it ,Thanks
and 
I'm new to open-source but eager to contribute. Could you offer some guidance on where to start? I can dedicate around an hour each day. Thank you.

Best regards,
Kanukollu Goutham

Comment (vowelparrot) [reply langchain/chains/conversational_retrieval/base.py:170]: This PR is helpful! I extended your PR in #3236 to update the - we're working on some larger improvements, for this to improve message passing, but it's great to be helping on the issues as you have done

Comment (kanukolluGVT) [reply langchain/chains/conversational_retrieval/base.py:170]: Thank you for the kind words! I'm glad that my pull request was helpful, and it's great to hear that you could build on it to make further improvements. I'm always happy to contribute to open-source projects and help others, so please feel free to reach out if you have issues I can help with.

**Pull request closed without merging**